bundle/tests/scorecard/config.yaml

```diff
@@ -45,6 +46,13 @@ stages:
     storage:
       spec:
         mountPath: {}
+  - image: quay.io/operator-framework/scorecard-test-kuttl:v2.0.0
+    labels:
+      suite: kuttlsuite
+      test: kuttltest1
```

Comment:
How is the label used? 

Should we use a more descriptive label?

Reply:
Afaics it's just printed in the scorecard output, i.e.

```
    {
      "kind": "Test",
      "apiVersion": "scorecard.operatorframework.io/v1alpha3",
      "spec": {
        "image": "quay.io/operator-framework/scorecard-test-kuttl:v2.0.0",
        "labels": {
          "suite": "kuttlsuite",
          "test": "kuttltest1"
        },
        "storage": {
          "spec": {
            "mountPath": {}
          }
        }
      },
      "status": {
        "results": [
          {
            "name": "smoketest",
            "state": "pass",
            "creationTimestamp": null
          }
        ]
      }
    }
```

Currently all tests of this kuttl suite (`bundle/tests/scorecard/kuttl`) are run in parallel as a single scorecard test. If we decide later that we want to split kuttl tests in multiple scorecard tests, we should add a descriptive label to each. For now I think it's fine, as in other projects we also run all kuttl tests in the same kuttl suite (i.e. `tests/e2e` is a single kuttle suite).